src/dumbo/src/tcp/endpoint.rs

```diff
@@ -312,7 +312,7 @@ fn build_response(http_version: Version, status_code: StatusCode, body: Body) ->
 
 /// Parses the request bytes and builds a `micro_http::Response` by the given callback function.
 fn parse_request_bytes(byte_stream: &[u8], callback: fn(Request) -> Response) -> Response {
-    let request = Request::try_from(byte_stream);
+    let request = Request::try_from(byte_stream, None);
```

Comment:
For `dumbo` we don't want to limit max request size?

Reply:
Not in this PR. In this PR we limit just PUT/PATCH requests, the response will be limited in a subsequent PR.

Reply:
This PR imposes a limit just for PUT/PATCH requests. AFAIK dumbo is used only for GET requests, the logic for GET requests will be handled in a subsequent PR.